ios/PolyPodApp/PolyPodUITests/UpdateNotificationTest.swift

```diff
@@ -60,6 +60,8 @@ class UpdateNotificationTest: XCTestCase {
             ]
         }
         app.launch()
+        let background = app.wait(for: .runningForeground, timeout: 30)
```

Comment:
Based on the error message, I am not sure this would help, but we can give it a try.

It seems that the app cannot be found on the simulator for a particular test.

Reply:
It's a long shot. It's not always happening, so I assume it's a timeout. If you look further down, it says it's not finding the PID for it. At any rate, waiting for the app to actually be running makes sense.

Reply:
So this is code we're not sure we need, that doesn't _look_ like we need it? I'd ask for a little comment here that explains the situation :D I know there's `git blame`, but still...

Reply:
Well, flappers are notably difficult to debug. I'd argue we _do_ need it, since we're just waiting for the application to actually be running before we do anything else with it, like seeing notifications.